deployment/kubernetes/values/values-dev.yaml

```diff
@@ -15,24 +15,31 @@ gethTxReplicas: 1
 gethMinerReplicas: 2
 gethMinerCpuLimit: 0.5
 
-# IPFS server
-ipfsImageTag: ipfs/go-ipfs
+# ipfs
+ipfsImage: ipfs/go-ipfs
 ipfsImageTag: master
 
+# ipfs-proxy
+ipfsProxyImage: ipfs-proxy
+ipfsProxyImageTag: 'e6d35c6'
+
+# origin-faucet
 faucetImage: origin-faucet
 faucetImageTag: '389fd1f'
 
+# event-listener
 eventlistenerImage: event-listener
 eventlistenerImageTag: 'f9a1525'
 eventlistenerWeb3Url: https://eth.dev.originprotocol.com/rpc
 
+# origin-discovery
 discoveryImage: origin-discovery
 discoveryImageTag: 'db88884'
 
-# IPFS server for OrbitDB
+# IPFS server for origin-messaging
```

Comment:
Is it worth keeping those values around even though we are not running this server ?

Reply:
I meant messagingIpfsImage and messagingIpfsImageTag - since we are not deploying this container anymore now that we are back to using the js-ipfs server...

Reply:
Oh yeah, duh! I will clean them up.